cmd/goal/clerk.go

```diff
@@ -97,6 +98,69 @@ var clerkCmd = &cobra.Command{
 	},
 }
 
+func waitForCommit(client libgoal.Client, txid string) {
+	if noWaitAfterSend {
```

Comment:
helper functions like this are easier to follow if you pass in the globals.  can you pass 'noWaitAfterSend' in as an arg?